### DISCUSSION TITLE
Someone managed to steal all my 27.5 XCH

### DISCUSSION THREAD

**Discussion by wetzelanton** [Chia-Network/chia-blockchain#14338] [Support]
Hello... 
Someone managed to steal all my 27.5 XCH on 07/01/2023.  I don't know how he was able to do that.

https://xchscan.com/address/xch1u6xyxrxez0pftu6wufnsqyf8vculdstgvqaa8dlhl547xcz0z4pqr35lsr

I've been farming Chia for almost a year and a half and never made a single transaction except the one I get as a reward from space pool. I don't know how it happened, but there are no transactions on my client, but when I check on https://xchscan.com, it's clear that someone with several transactions took out all the balance from my wallet. How did it happen?
I use Windows 10, I have a ESET internet security.
I can clearly see my Chias on his address and I cannot do anything :( you cant imagine my pain right now. 
Please help me!
THANK YOU

This is the address of the the thief:

xch1mgda9ll4kst0fh30847tdurjwq9rj6vxumqhxu3xf6r2xt864hdq8s99ut

**Comment by Laushd** [comment]
Hello. 27.45 XCH was stolen from me on March 7th. The official Chia wallet. It is almost always turned off. The last phrase from him is only in my paper notebook. Stolen in two transactions, 27XCH and 0.45 XCH. Can you help me get them back?

Address of the thief:  xch1mgda9ll4kst0fh30847tdurjwq9rj6vxumqhxu3xf6r2xt864hdq8s99ut

Confirmed at Height | 3346415
and 3346429

I have been saving these coins for many years, and they were stolen.

Please help punish the thief and return the coins. 
Thanks! I am waiting for your answer! 
Dmitry.

**Reply by BrandtH22** [reply]
Hey @Laushd , 
Since chia is a fully decentralized blockchain there is unfortunately no one with the ability to reclaim the coins.

We highly urge you to review our blog article about securing your funds and making yourself a hard target:
https://www.chia.net/2021/05/28/securing-your-chia-how-to-be-a-hard-target/

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
